spec/features/routing/rate_group_duplicators/new_rate_group_duplicator_spec.rb

```diff
@@ -24,5 +24,5 @@
   end
 
   include_examples :changes_records_qty_of, Routing::RateGroup, by: 1
-  include_examples :shows_flash_message, :notice, 'Rate group duplicator was successfully created.'
+  include_examples :shows_flash_message, :notice, 'Rate group duplicator form was successfully created.'
```

Comment:
message should stay the same

renaming of resource class should not affect such messages

to fix this you need to add custom translation for class to `config/locales/en.yml`

```
  activemodel:
    models:
      routing/rate_group_duplicator_form: "Rate Group Duplicator"
``

do this for other resource classes that you've changed too